src/nodes.coffee

```diff
@@ -1456,6 +1461,18 @@ exports.Class = class Class extends Base
 
     ctor
 
+  proxyBoundMethods: (o) ->
+    @variable ?= new IdentifierLiteral o.scope.freeVariable '_class' if @parent
+    [@variable, @variableRef] = @variable.cache o unless @variableRef?
```

Comment:
This needs to account for `@variable` being null if there's no `@parent`, e.g.

```coffee
class
  a: =>
```

currently throws "Cannot read property 'cache' of null".